scripts/generate_readmes.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+crates_to_document=("hdk" "holochain_keystore")
+
+for crate in "${crates_to_document[@]}"; do
+    echo 'generating README for crate' "$crate";
+    cargo readme -r crates/"$crate" -o README.md
+done
+
+# have any READMEs been updated?
+git diff --exit-code --quiet
+readmes_updated=$?
+if [[ "$readmes_updated" == 1 ]]; then
+    echo 'READMEs have been updated, committing changes'
+    git config user.name release-ci
+    git config user.email ci@holo.host
+    git commit -am "docs(crate-level): generated READMEs from doc comments"
+    git push origin develop
```

Comment:
within the release workflow we have to be careful about timing the pushes and what branch we're on. the "finalize" step in the workflow takes care of that and so a commit is all we should need here.

```suggestion
```

to make that work we need to make the changes in either in the prepare step which will store the holochain repo in the github cache, which will later be retrieved in the finalized step. or we make the modifications in the finalize step after the repo has been retrieved.

Reply:
I saw the option to do it in the prepare step, but then got entangled into passing env vars and kept it simple by committing and pushing to develop.

Reply:
I moved the one relevant command to a step in `release-prepare.yml`.